lib/school.rb

```diff
@@ -0,0 +1,35 @@
+class School
+  attr_reader:start_time,
```

Comment:
Nitpick but you probably want a space here between attr_reader and your attribute